sdk-actors/src/main/java/io/dapr/actors/runtime/ActorRuntime.java

```diff
@@ -112,29 +112,37 @@ public static ActorRuntime getInstance() {
    * Registers an actor with the runtime.
    *
    * @param clazz            The type of actor.
-   * @param objectSerializer Serializer for Actor's state and transient objects.
+   * @param objectSerializer Serializer for Actor's transient objects.
+   * @param stateSerializer  Serializer for Actor's state objects.
    * @param <T>              Actor class type.
    */
-  public <T extends AbstractActor> void registerActor(Class<T> clazz, DaprObjectSerializer objectSerializer) {
-    registerActor(clazz, null, objectSerializer);
+  public <T extends AbstractActor> void registerActor(
+    Class<T> clazz, DaprObjectSerializer objectSerializer, DaprObjectSerializer stateSerializer) {
+    registerActor(clazz, null, objectSerializer, stateSerializer);
   }
 
   /**
    * Registers an actor with the runtime.
    *
    * @param clazz            The type of actor.
    * @param actorFactory     An optional factory to create actors. This can be used for dependency injection.
-   * @param serializer       Serializer for Actor's state and transient objects.
+   * @param serializer       Serializer for Actor's transient objects.
+   * @param stateSerializer  Serializer for Actor's state objects.
    * @param <T>              Actor class type.
    */
   public <T extends AbstractActor> void registerActor(
-    Class<T> clazz, ActorFactory<T> actorFactory, DaprObjectSerializer serializer) {
+    Class<T> clazz, ActorFactory<T> actorFactory,
+    DaprObjectSerializer serializer,
```

Comment:
Nit: can we rename `serializer` to be more specific name.

Reply:
I am not sure what a better name would be. Let me find something and change it.

Reply:
See the latest version, please.